commands/mentor.py

```diff
@@ -1,66 +1,388 @@
+        "helpee_helpme_complete": "Your mentor has marked this HelpMe request as complete. We'd really appreciate it if you could fill out this survey to let us know how it went: {}",
+        "mentor_helpme_complete": "XP deposited! Thanks for helping out :)",
+        "log_helpme_complete": "{} completed a HelpMe request from {}",
+        "helpee_delay": "Sorry, there's been a slight delay in finding an available mentor to help. We'll get someone to contact you as soon as possible",
+        "mentor_declined": "You have declined this HelpMe request so it will be passed to another mentor",
+        "mentor_escalated": "This HelpMe request has been escalated to DTesters mods",
+        "mentor_claim_failed": "{} Your DMs are closed so you can't claim this request"
+    }
+    emoji = {
+        "complete": {
+            "name": "greenTick",
+            "id": 312314752711786497
+        },
+        "decline": {
+            "name": "redTick",
+            "id": 312314733816709120
+        },
+        "escalate": {
+            "name": "discordPolice",
+            "id": 233444108658671616
+        },
+        "mentor": {
+            "name": "dabPingWordless",
+            "id": 456143314404769793
+        }
+    }
+    join_phrase = "to the Bug Hunters:tm:!"
+    mentor_role_id = 502115003445411840
+    helpme_embed_desc = ["You've been selected to assist with the following HelpMe request",
+                         "",
+                         "Please DM the user directly to assist and then react as necessary once complete:",
+                         "<:{}> - I have finished helping the user",
+                         "<:{}> - I am unavailable. Please pass this request to another mentor",
+                         "<:{}> - This request needs to be escalated to DTesters mods"]
+    survey_link = 'https://dabbit.typeform.com/to/mnlaDU?id={}'
+    helpme_xp = 1
 
 
 @Plugin.with_config(MentorConfig)
 class MentorPlugin(Plugin):
-    def get_avail_mentors(self):
-        return [u.user.id for u in self.bot.client.state.guilds.get(197038439483310086).members.values() if self.config.MENTOR_ID in u.roles and hasattr(u.user.presence, "status") and str(u.user.presence.status) == "online"]
 
-    def ping_mentor(self, mentor, author, content):
-        self.bot.client.api.channels_messages_create(self.config.MENTOR_CHANNEL, (self.config.HELP_MESSAGE.format(mentor, author, content)))
+    def load(self, ctx):
+        super(MentorPlugin, self).load(ctx)
+        self.client = MongoClient(self.config.mongodb_host, self.config.mongodb_port,
+                                  username=self.config.mongodb_username,
+                                  password=self.config.mongodb_password)
+        self.helpme = self.client.mentors.helpme
+        self.users = self.client.experience.users
+
+    def unload(self, ctx):
+        self.helpme.save()
+        self.users.save()
+        super(MentorPlugin, self).unload(ctx)
```

Comment:
can you just do 
```suggestion
        super().unload(ctx)
```
instead? this is python3, while this works during loading, for some reason using the python2 syntax makes this error out when trying to reload the plugin

Reply:
Yep, done (also changed the load call)